kubernetes-membership-scheme/src/main/java/org/wso2/carbon/membership/scheme/kubernetes/KubernetesMembershipScheme.java

```diff
@@ -139,6 +139,11 @@ private Parameter getParameter(String name) {
 
         @Override public void memberAdded(MembershipEvent membershipEvent) {
             Member member = membershipEvent.getMember();
+            List<String> memberList = nwConfig.getJoin().getTcpIpConfig().setEnabled(true).getMembers();
```

Comment:
.setEnabled(true) is not needed. 

TcpIpConfig tcpIpConfig = nwConfig.getJoin().getTcpIpConfig();
List<String> memberList = tcpIpConfig.getMembers();

Reply:
fixed with 7913ee0074c4ebbc8590adf1013ffb6eea5efee6